.circleci/workflows.yml

```diff
@@ -3,7 +3,7 @@ parameters:
   ManualBuild:
     description: 'When `true`, pipeline will continue to build even if its been filtered out by config.yml (Will Not Run on Master Branch)'
     type: boolean
-    default: false
+    default: true
```

Comment:
@gweinjc @jworkmanjc Should this still be set to true?

Reply:
Sounds like this shouldn't matter. Won't run on `master` branch anyway so we should be good to go.